### PR TITLE
Update branch

### PR DESCRIPTION
**Pull request by priyatharsan**
fixes for: https://github.com/Aifred-Health/Vulcan2/issues/18

### CODE REVIEW

File: vulcanai2/models/basenetwork.py [hunk 1]
```diff
@@ -284,20 +288,18 @@ def _init_optimizer(self, optim_spec):
         return optim_class(self.parameters(), **optim_spec)
 
     @staticmethod
-    def _get_criterion(criterion_spec):
-        criterion_class = getattr(loss, criterion_spec["name"])
-        criterion_spec = pdash.omit(criterion_spec, "name")
-        return criterion_class(**criterion_spec)
+    def _init_criterion(criterion_spec):
+        return criterion_spec()
```
Comment (nirtiac) [inline vulcanai2/models/basenetwork.py]: Cool 👍

File: vulcanai2/models/multinn.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+from .basenetwork import BaseNetwork
+from .layers import DenseUnit, ConvUnit
+
+import numpy as np
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class MultiInputNN(BaseNetwork, nn.Module):
```
Comment (rfratila) [inline vulcanai2/models/multinn.py]: No it needs to be a part of ConvNet and DenseNet

File: vulcanai2/plotters/visualization.py [hunk 3]
```diff
@@ -148,6 +149,34 @@ def _plot_reduction(x_transform, train_y, label_map, title='Dim Reduction'):
     plt.title(title)
     plt.show(False)
 
+def display_confusion_matrix(cm, classes):
```
Comment (rfratila) [inline vulcanai2/plotters/visualization.py]: class_list is a more intuitive name than classes

File: vulcanai2/models/multinn.py (hunk 2 above)
Comment (rfratila) [reply vulcanai2/models/multinn.py]: would be too burdensome to have this be in-between each ensemble tbh

**Pull request merged**